src/topic/TopicMessageQuery.js

```diff
@@ -115,16 +116,18 @@ export default class TopicMessageQuery {
                         error.toString().includes("NOT_FOUND") ||
                         error.toString().includes("UNAVAILABLE") ||
                         error.toString().includes("RESOURCE_EXHAUSTED") ||
-                        error.toString().includes("INTERNAL")
+                        (error.toString().includes("INTERNAL") &&
```

Comment:
Why would there be a non-gRPC `StatusObject` that includes gRPC status codes? It seems like errors in this if statement would more likely to be lower level HTTP2 errors, right? In which case you might want to just retry all if so.

Reply:
This really isn't part of this PR but I do agree that this whole part of the default retry handler should likely be thrown out and we should test for a "gRPC error" more reliably somehow.

Reply:
Not entirely sure on what you're suggesting. I did remove that block of code and just `return true`, but I could be misunderstanding what you want me to do here.